active/TabView/TabView_v2.md

```diff
@@ -0,0 +1,142 @@
+    <muxc:TabView.TabItems>
+    
+        <muxc:TabViewItem Header="Events" >
+            <muxc:TabViewItem.IconSource>
+                <muxc:SymbolIconSource Symbol="Calendar" />
+            </muxc:TabViewItem.IconSource>
+                    
+            <!-- ... -->
+                    
+        </muxc:TabViewItem>
+
+        <muxc:TabViewItem Header="Messages">
+            <muxc:TabViewItem.IconSource>
+                <muxc:SymbolIconSource Symbol="Message" />
+            </muxc:TabViewItem.IconSource>
+
+            <!-- ... -->
+
+        </muxc:TabViewItem>
+    </muxc:TabView.TabItems>
+</muxc:TabView>
+```
+
+## TabView.CloseButtonOverlayMode
+
+A TabView where the unselected tab items have a close button that only displays when the mouse is over it. (The selected tab item always has a close button, regardless of this property value.)
+
+```XML
+<muxc:TabView CloseButtonOverlayMode="OnPointerOver">
+    <!-- ... -->
+</muxc:TabView>
+```
+
+# API Notes
+
+## TabViewWidthMode: Compact
+
+This change introduces a new value to the existing `TabViewWidthMode` enum: `Compact`.
+
+Setting `Compact` will cause unselected tab headers to display only the tab's icon. If an icon is not set, the tab will be the same size as if an icon was present but display an empty space where the icon would be.
+
+The selected tab will render at its natural size (ie. the size it would render in `SizeToContent` mode.)
+
+![Compact](./TabView_Width_Compact.png)
+
+## CloseButtonOverlayMode
+
+Describes the behavior of the x-to-close button found on each TabViewItem. 
+
+The `CloseButtonOverlayMode` enum has three values: {`Auto`, `OnPointerOver`, `Always`}
+
+> The `CloseButtonOverlayMode` will only affect tabs that are closeable - ie. the value of the `TabViewItem`'s `IsClosable` property is `TRUE`.
+
+`OnPointerOver`: The x-to-close button only appears on the selected tab and any tabs that are being hovered.
+
+![Hover](./TabView_Close_Hover.gif)
+
+`Always`: The x-to-close button always appears on every tab. 
+
+![Persistent](./TabView_Close_Persistent.png)
+
+`Auto`: Currently maps to `Always`.
+
+> Prior to this change, the behavior of the x-to-close button was `Always` with no option to change the behavior.
+
+> See the [Appendix](#Appendix) section for why this value is an enum and not a boolean.
+
+# API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+```
+[WUXC_VERSION_MUXONLY]
+[webhosthidden]
+enum TabViewWidthMode
+{
+    Equal = 0,
+    SizeToContent = 1,
+    Compact = 2,
+};
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+enum TabViewCloseButtonOverlayMode
+{
+    Auto = 0,
+    OnPointerOver = 1,
+    Always = 2,
+};
+
+[WUXC_VERSION_MUXONLY]
+[webhosthidden]
+[contentproperty("TabItems")]
+unsealed runtimeclass TabView : Windows.UI.Xaml.Controls.Control
+{
+
+    [WUXC_VERSION_PREVIEW]
+    [MUX_DEFAULT_VALUE("winrt::TabViewCloseButtonOverlayMode::Always")]
```

Comment:
Shouldn't the default be `Auto`, not `Always`? Otherwise, what's the point of `Auto`?

Reply:
Yes, `Auto` should be the default